samples/storageclass/storageclass.yaml

```diff
@@ -33,6 +33,11 @@ parameters:
   # Optional: true
   # Uncomment the line below if you want to use mkfsFormatOption
   # mkfsFormatOption: "<mkfs_format_option>" # Insert file system format option
+  # Filesytem type for volumes created by storageclass
+  # Default value: None if defaultFsType is not mentioned in values.yaml
+  # Else defaultFsType value mentioned in values.yaml
+  # will be used as default value
+  csi.storage.k8s.io/fstype: xfs
```

Comment:
why is fsType set to xfs in this SC, should it not be ext4? My understanding is storageclass-xfs is for xfs and storageclass is for ext4. @JacobGros @abhi16394 @jooseppi-luna  - could you guys please check this?

Reply:
Yes, second what Bharath says on this.

Reply:
Addressed